### PR TITLE
translation contribution guide

### PR DESCRIPTION
**Pull request by ElisaCapololo**
This PR is a translation work

## Changes

- translating contribution guide into Portuguese issue  #88 

## Screenshots

guide

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

Fixes #

### CODE REVIEW

**Comment by aosccode** [comment]
Fixes #88

File: Translations/Portuguese/Basic Information.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
 This is the link  to the slack bot wiki basic information page to be translated.
```
Comment (aosccode) [inline Translations/Portuguese/Basic Information.md:1]: I don't think we need this line here :)

File: Translations/Portuguese/Basic Information.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
-[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
\ No newline at end of file
+[Basic Information](https://github.com/chaoss/chaoss-slack-bot/wiki/Basic-Information)
+
+> ### Informação básica
```
Comment (aosccode) [inline Translations/Portuguese/Basic Information.md:4]: Should it be `Informações básicas`?

File: Translations/Portuguese/Basic Information.md [hunk 3]
```diff
@@ -1,2 +1,20 @@
...
+
+> #### Funções do bot
```
Comment (aosccode) [inline Translations/Portuguese/Basic Information.md:6]: It should be `Funcionalidades do bot` in IMHO :)

File: Translations/Portuguese/Getting Started With Bolt For Javascript.md
Comment (aosccode) [inline Translations/Portuguese/Getting Started With Bolt For Javascript.md:1]: You are fixing more than one issue, you should mention that.

I see this PR fixes #90, #85, #83, and #88 

Either you use this PR to fix all the issues, or open a new PR for each issue. 
I am not sure fixing all four issues in one PR will be counted as 4 PRs. Please choose wisely :)

File: Translations/Portuguese/Contribution Guide.md [hunk 4]
```diff
@@ -1,2 +1,63 @@
...
+
+## Abrindo um problema
+
+Se você estiver enfrentando problemas com o CHAOSS Slack Bot ou tiver alguma dúvida que gostaria de ajuda para responder, sinta-se à vontade para abrir um [issue](https://github.com/chaoss/chaoss-slack-bot/issues). Para nos ajudar a evitar duplicatas, pedimos que você pesquise brevemente o seu problema ou dúvida em nossas [issues](https://github.com/chaoss/chaoss-slack-bot/issues) antes de abrir uma nova. Observe que se você abrir um relatório de bug, não poderemos ajudá-lo até que você nos forneça todas as informações relevantes. Respeitosamente, não temos tempo para tentar recriar um erro fornecido com o mínimo ou nenhum contexto, portanto, ao fornecer essas informações, você está nos ajudando a ajudá-lo! Forneça descrições da melhor maneira possível e inclua capturas de tela e erroros de logs, se aplicável.
```
Comment (aosccode) [inline Translations/Portuguese/Contribution Guide.md:10]: It should be nice if we have one line per sentence. 
@Busayo-ojo do you have any thoughts?

File: Translations/Portuguese/Contribution Guide.md [hunk 5]
```diff
@@ -1,2 +1,63 @@
...
+
+6. Confirme as alterações com uma mensagem de commit descritiva (aqui está um [guia](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) para escrever boas mensagens de commit) e envie para seu fork. PS: se você não estiver familiarizado com o DCO, leia o guia sobre como signing off commits abaixo antes de voltar a esta etapa.
+```
+$ git add .
+$ git commit -s -m "descriptive commit message"
+$ git push -u origin my-new-branch
+```
+6. Envie uma pull request. Neste ponto, você está esperando por nós. Gostamos de comentar pelo menos as solicitações pull request dentro de três dias úteis (e, normalmente, um dia útil). Assim que um de nossos mantenedores tiver a chance de revisar seu PR, iremos marcá-lo como "revisão necessária" e fornecer feedback específico sobre suas alterações ou prosseguiremos e concluiremos a solicitação pull request.
+
+## Signing-off em Commits
```
Comment (aosccode) [inline Translations/Portuguese/Contribution Guide.md:40]: Adicione assinatura nos commits :)

**Review by aosccode — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution and your hard work @ElisaCapololo. I just left a couple of comm

**Comment by aosccode** [comment]
I have seen that #85 has being fixed by #92 and it was submitted earlier than this PR :)

**Comment by antonio-pedro99** [comment]
Hi @ElisaCapololo, can you remove the translation of the Contributing Guide from this PR? It is being fixed by #85 already
BTW, are you addressing my comments?

File: Translations/Portuguese/Getting Started With Bolt For Javascript.md
Comment (antonio-pedro99) [reply Translations/Portuguese/Getting Started With Bolt For Javascript.md:1]: As this PR is solving what #92 is already fixing, and other changes are unrelated to the issue, I think we should close this PR.
CC: @Busayo-ojo

**Comment by Busayo-ojo** [comment]
Thanks for working on this @ElisaCapololo!

**Pull request merged**